Traffic_Lights/Traffic_Lights/Traffic_Lights/main.asm

```diff
@@ -0,0 +1,170 @@
+;
+; Traffic_Lights.asm
+;
+; Created: 28/11/2018 12:48:16
+; Author : Lukas Zagata, Daniela Porcescu
+;
+
+
+; Replace with your application code
+START:
+	
+	; Setup DDR for LEDs
+	LDI R24, (1 << PA0)
```

Comment:
i like that you left shift a bit instead of just writing the bit pattern directly